jupyterhub/utils.py

```diff
@@ -295,6 +295,8 @@ async def is_reachable():
                 errno.ECONNRESET,
             }:
                 app_log.warning("Failed to connect to %s (%s)", url, e)
+        except ValueError as e:
```

Comment:
We keep expanding this, I wonder if this should be `except Exception as e`.  That would swallow some errors inappropriately, e.g. `NameError: url` if there was a typo there, but I think the _vast_ majority of errors to be raised here are likely to be like the one you just encountered. And the failure mode of catching too many here is only a delay in reporting real errors, rather than the current situation which is inappropriate failure that can't be worked around.

What do you think?

Reply:
If one can trust the `Spawner.poll` function, errors and non-running notebook-servers should be catched anyway, shouldn't they?
IMHO we can change it to `except Exception as e`. 

I can update the PR if you like

Reply:
Yeah, go for it. I think that makes the most sense.